--- international/engine/battle/move_effects/haze.asm
+++ japanese/engine/battle/move_effects/haze.asm
@@ -1,9 +1,9 @@
 HazeEffect_:
 	ld a, $7
 ; store 7 on every stat mod
-	ld hl, wPlayerMonAttackMod
+	ld hl, wPlayerMonStatMods
 	call ResetStatMods
-	ld hl, wEnemyMonAttackMod
+	ld hl, wEnemyMonStatMods
 	call ResetStatMods
 ; copy unmodified stats to battle stats
 	ld hl, wPlayerMonUnmodifiedAttack
@@ -36,7 +36,7 @@
 	ld [wEnemyDisabledMove], a
 	ld hl, wPlayerDisabledMoveNumber
 	ld [hli], a
-	ld [hl], a
+	ld [hl], a ; wEnemyDisabledMoveNumber
 	ld hl, wPlayerBattleStatus1
 	call CureVolatileStatuses
 	ld hl, wEnemyBattleStatus1
@@ -48,13 +48,14 @@
 
 CureVolatileStatuses:
 	res CONFUSED, [hl]
-	inc hl ; BATTSTATUS2
-	ld a, [hl]
+	inc hl
+	ld a, [hl] ; wPlayerBattleStatus2 or wEnemyBattleStatus2
 	; clear USING_X_ACCURACY, PROTECTED_BY_MIST, GETTING_PUMPED, and SEEDED statuses
 	and ~((1 << USING_X_ACCURACY) | (1 << PROTECTED_BY_MIST) | (1 << GETTING_PUMPED) | (1 << SEEDED))
-	ld [hli], a ; BATTSTATUS3
-	ld a, [hl]
-	and %11110000 | (1 << TRANSFORMED) ; clear Bad Poison, Reflect and Light Screen statuses
+	ld [hli], a
+	ld a, [hl] ; wPlayerBattleStatus3 or wEnemyBattleStatus3
+	; clear Bad Poison, Reflect and Light Screen statuses
+	and ~((1 << BADLY_POISONED) | (1 << HAS_LIGHT_SCREEN_UP) | (1 << HAS_REFLECT_UP))
 	ld [hl], a
 	ret
 
@@ -77,5 +78,6 @@
 	ret
 
 StatusChangesEliminatedText:
-	text_far _StatusChangesEliminatedText
-	text_end
+	text "すべての　ステータスが"
+	line "もとに　もどった！"
+	prompt
