--- international/engine/battle/move_effects/recoil.asm
+++ japanese/engine/battle/move_effects/recoil.asm
@@ -27,7 +27,7 @@
 .updateHP
 ; subtract HP from user due to the recoil damage
 	ld a, [hli]
-	ld [wHPBarMaxHP+1], a
+	ld [wHPBarMaxHP + 1], a
 	ld a, [hl]
 	ld [wHPBarMaxHP], a
 	push bc
@@ -40,10 +40,10 @@
 	ld [hld], a
 	ld [wHPBarNewHP], a
 	ld a, [hl]
-	ld [wHPBarOldHP+1], a
+	ld [wHPBarOldHP + 1], a
 	sbc b
 	ld [hl], a
-	ld [wHPBarNewHP+1], a
+	ld [wHPBarNewHP + 1], a
 	jr nc, .getHPBarCoords
 ; if recoil damage is higher than the Pokemon's HP, set its HP to 0
 	xor a
@@ -65,6 +65,8 @@
 	predef UpdateHPBar2
 	ld hl, HitWithRecoilText
 	jp PrintText
+
 HitWithRecoilText:
-	text_far _HitWithRecoilText
-	text_end
+	text "<USER>は　こうげきの"
+	line "はんどうを　うけた！"
+	prompt
