--- international/engine/battle/wild_encounters.asm
+++ japanese/engine/battle/wild_encounters.asm
@@ -25,7 +25,6 @@
 .next
 ; determine if wild pokemon can appear in the half-block we're standing in
 ; is the bottom left tile (8,9) of the half-block we're standing in a grass/water tile?
-; note that by using the bottom left tile, this prevents the "left-shore" tiles from generating grass encounters
 	hlcoord 8, 9
 	ld c, [hl]
 	ld a, [wGrassTile]
