--- international/engine/gfx/hp_bar.asm
+++ japanese/engine/gfx/hp_bar.asm
@@ -21,22 +21,22 @@
 	rr e
 	srl d
 	rr e
-	ldh a, [hMultiplicand+1]
-	ld b, a
-	ldh a, [hMultiplicand+2]
+	ldh a, [hMultiplicand + 1]
+	ld b, a
+	ldh a, [hMultiplicand + 2]
 	srl b              ; divide multiplication result as well
 	rr a
 	srl b
 	rr a
-	ldh [hMultiplicand+2], a
-	ld a, b
-	ldh [hMultiplicand+1], a
+	ldh [hMultiplicand + 2], a
+	ld a, b
+	ldh [hMultiplicand + 1], a
 .maxHPSmaller256
 	ld a, e
 	ldh [hDivisor], a
 	ld b, $4
 	call Divide
-	ldh a, [hMultiplicand+2]
+	ldh a, [hMultiplicand + 2]
 	ld e, a            ; e = bc * 48 / de (num of pixels of HP bar)
 	pop hl
 	and a
@@ -61,7 +61,7 @@
 	push bc
 	call UpdateHPBar_CalcHPDifference
 	ld a, e
-	ld [wHPBarHPDifference+1], a
+	ld [wHPBarHPDifference + 1], a
 	ld a, d
 	ld [wHPBarHPDifference], a
 	pop bc
@@ -76,13 +76,13 @@
 	call GetPredefRegisters
 	ld a, [wHPBarNewHP]
 	ld e, a
-	ld a, [wHPBarNewHP+1]
+	ld a, [wHPBarNewHP + 1]
 	ld d, a
 .animateHPBarLoop
 	push de
 	ld a, [wHPBarOldHP]
 	ld c, a
-	ld a, [wHPBarOldHP+1]
+	ld a, [wHPBarOldHP + 1]
 	ld b, a
 	call UpdateHPBar_CompareNewHPToOldHP
 	jr z, .animateHPBarDone
@@ -92,7 +92,7 @@
 	ld a, c
 	ld [wHPBarNewHP], a
 	ld a, b
-	ld [wHPBarNewHP+1], a
+	ld [wHPBarNewHP + 1], a
 	call UpdateHPBar_CalcOldNewHPBarPixels
 	ld a, e
 	sub d         ; calc pixel difference
@@ -102,7 +102,7 @@
 	ld a, c
 	ld [wHPBarNewHP], a
 	ld a, b
-	ld [wHPBarNewHP+1], a
+	ld [wHPBarNewHP + 1], a
 	call UpdateHPBar_CalcOldNewHPBarPixels
 	ld a, d
 	sub e         ; calc pixel difference
@@ -114,8 +114,8 @@
 .noPixelDifference
 	ld a, [wHPBarNewHP]
 	ld [wHPBarOldHP], a
-	ld a, [wHPBarNewHP+1]
-	ld [wHPBarOldHP+1], a
+	ld a, [wHPBarNewHP + 1]
+	ld [wHPBarOldHP + 1], a
 	pop de
 	jr .animateHPBarLoop
 .animateHPBarDone
@@ -123,7 +123,7 @@
 	ld a, e
 	ld [wHPBarOldHP], a
 	ld a, d
-	ld [wHPBarOldHP+1], a
+	ld [wHPBarOldHP + 1], a
 	or e
 	jr z, .monFainted
 	call UpdateHPBar_CalcOldNewHPBarPixels
@@ -214,14 +214,9 @@
 	ld [wHPBarTempHP], a
 	push hl
 	ld de, $15
-	ldh a, [hUILayoutFlags]
-	bit BIT_PARTY_MENU_HP_BAR, a
-	jr z, .hpBelowBar
-	ld de, $9
-.hpBelowBar
 	add hl, de
 	push hl
-	ld a, ' '
+	ld a, '　'
 	ld [hli], a
 	ld [hli], a
 	ld [hli], a
