--- international/engine/overworld/cut.asm
+++ japanese/engine/overworld/cut.asm
@@ -22,8 +22,9 @@
 	jp PrintText
 
 .NothingToCutText
-	text_far _NothingToCutText
-	text_end
+	text "いあいぎりで"
+	line "きれそうな　ものが　ありません！"
+	prompt
 
 .canCut
 	ld [wCutTile], a
@@ -68,15 +69,16 @@
 	jp RedrawMapView
 
 UsedCutText:
-	text_far _UsedCutText
-	text_end
+	text_ram wNameBuffer
+	text "は"
+	line "いあいぎりで　きりまくった！"
+	prompt
 
 InitCutAnimOAM:
 	xor a
 	ld [wWhichAnimationOffsets], a
 	ld a, %11100100
 	ldh [rOBP1], a
-	call UpdateCGBPal_OBP1
 	ld a, [wCutTile]
 	cp $52
 	jr z, .grass
@@ -125,10 +127,10 @@
 
 .OAMBlock:
 ; tile ID, attributes
-	db $fc, OAM_PAL1 | OAM_HIGH_PALS
-	db $fd, OAM_PAL1 | OAM_HIGH_PALS
-	db $fe, OAM_PAL1 | OAM_HIGH_PALS
-	db $ff, OAM_PAL1 | OAM_HIGH_PALS
+	db $fc, OAM_PAL1
+	db $fd, OAM_PAL1
+	db $fe, OAM_PAL1
+	db $ff, OAM_PAL1
 
 GetCutOrBoulderDustAnimationOffsets:
 	ld hl, wSpritePlayerStateData1YPixels
