--- international/engine/items/item_effects.asm
+++ japanese/engine/items/item_effects.asm
@@ -175,6 +175,7 @@
 	ld b, $63
 	jp nz, .setAnimData
 	jp .captured
+
 .notOldManBattle
 ; If the player is fighting the ghost Marowak, set the value that indicates the
 ; Pokémon can't be caught and skip the capture calculations.
@@ -293,7 +294,6 @@
 	inc a
 
 .skip2
-
 ; Let W = ((MaxHP * 255) / BallFactor) / max(HP / 4, 1). Calculate W.
 	ldh [hDivisor], a
 	ld b, 4
@@ -361,7 +361,6 @@
 	jr z, .skip4
 
 .skip4
-
 ; Let Y = (CatchRate * 100) / BallFactor2. Calculate Y.
 	ld a, b
 	ldh [hDivisor], a
@@ -530,6 +529,7 @@
 	jp z, .oldManCaughtMon ; if so, don't give the player the caught Pokémon
 	cp BATTLE_TYPE_PIKACHU
 	jr z, .oldManCaughtMon ; same with Pikachu battle
+
 	ld hl, ItemUseBallText05
 	call PrintText
 
@@ -562,7 +562,7 @@
 
 .skipShowingPokedexData
 	ld a, $1
-	ld [wPikachuEmotionModifier], a
+	ld [wd49b], a
 	ld a, $85
 	ld [wPikachuMood], a
 	ld a, [wPartyCount]
@@ -611,44 +611,65 @@
 ItemUseBallText00:
 ;"It dodged the thrown ball!"
 ;"This pokemon can't be caught"
-	text_far _ItemUseBallText00
-	text_end
+	text "よけられた！"
+	line "こいつは　つかまりそうにないぞ！"
+	prompt
+
 ItemUseBallText01:
 ;"You missed the pokemon!"
-	text_far _ItemUseBallText01
-	text_end
+	text "#に"
+	line "うまく　あたらなかった！"
+	prompt
+
 ItemUseBallText02:
 ;"Darn! The pokemon broke free!"
-	text_far _ItemUseBallText02
-	text_end
+	text "ざんねん！　#が"
+	line "ボールから　でてしまった！"
+	prompt
+
 ItemUseBallText03:
 ;"Aww! It appeared to be caught!"
-	text_far _ItemUseBallText03
-	text_end
+	text "ああ！"
+	line "つかまえたと　おもったのに！"
+	prompt
+
 ItemUseBallText04:
 ;"Shoot! It was so close too!"
-	text_far _ItemUseBallText04
-	text_end
+	text "おしい！"
+	line "あと　ちょっとの　ところだったのに！"
+	prompt
+
 ItemUseBallText05:
 ;"All right! {MonName} was caught!"
 ;play sound
-	text_far _ItemUseBallText05
+	text "やったー！"
+	line "@"
+	text_ram wEnemyMonNick
+	text "を　つかまえたぞ！@"
 	sound_caught_mon
 	text_promptbutton
 	text_end
+
 ItemUseBallText07:
 ;"X was transferred to Bill's PC"
-	text_far _ItemUseBallText07
-	text_end
+	text_ram wBoxMonNicks
+	text "は　マサキの　ところへ"
+	line "てんそうされた！"
+	prompt
+
 ItemUseBallText08:
 ;"X was transferred to someone's PC"
-	text_far _ItemUseBallText08
-	text_end
+	text_ram wBoxMonNicks
+	text "は　だれかの　<PC>に"
+	line "てんそうされた！"
+	prompt
 
 ItemUseBallText06:
 ;"New DEX data will be added..."
 ;play sound
-	text_far _ItemUseBallText06
+	text_ram wEnemyMonNick
+	text "の　データが　あたらしく"
+	line "#ずかんに　セーブされます！@"
 	sound_dex_page_added
 	text_promptbutton
 	text_end
@@ -751,7 +772,6 @@
 	dec a
 	ld [wJoyIgnore], a
 	call PlayDefaultMusic ; play walking music
-	call GBPalWhiteOutWithDelay3
 	jp LoadWalkingPlayerSpriteGraphics
 
 ; uses a simulated button press to make the player move forward
@@ -777,12 +797,14 @@
 	ret
 
 SurfingGotOnText:
-	text_far _SurfingGotOnText
-	text_end
+	text "<PLAYER>は　@"
+	text_ram wNameBuffer
+	text "にのった！"
+	prompt
 
 SurfingNoPlaceToGetOffText:
-	text_far _SurfingNoPlaceToGetOffText
-	text_end
+	text "おりる　ばしょが　ない！"
+	prompt
 
 ItemUsePokedex:
 	predef_jump ShowPokedexMenu
@@ -819,7 +841,7 @@
 	ld hl, RefusingText
 	call PrintText
 	ld a, $4
-	ld [wPikachuEmotionModifier], a
+	ld [wd49b], a
 	ld a, $82
 	ld [wPikachuMood], a
 	jr .canceledItemUse
@@ -892,8 +914,9 @@
 	ret
 
 RefusingText:
-	text_far _RefusingText
-	text_end
+	text_ram wNameBuffer
+	text "は　いやがっている！"
+	prompt
 
 ItemUseVitamin:
 	ld a, [wIsInBattle]
@@ -1015,7 +1038,7 @@
 	inc hl ; hl = address of current HP
 	ld a, [hli]
 	ld b, a
-	ld [wHPBarOldHP+1], a
+	ld [wHPBarOldHP + 1], a
 	ld a, [hl]
 	ld c, a
 	ld [wHPBarOldHP], a ; current HP stored at wHPBarOldHP (2 bytes, big-endian)
@@ -1114,7 +1137,7 @@
 	ld bc, MON_MAXHP - (MON_HP + 1)
 	add hl, bc ; hl now points to max HP
 	ld a, [hli]
-	ld [wHPBarMaxHP+1], a
+	ld [wHPBarMaxHP + 1], a
 	ld a, [hl]
 	ld [wHPBarMaxHP], a ; max HP stored at wHPBarMaxHP (2 bytes, big-endian)
 	ld a, [wPseudoItemID]
@@ -1158,25 +1181,19 @@
 	ldh a, [hQuotient + 2]
 	ld b, a
 	ld a, [hl]
-	ld [wHPBarOldHP+1], a
+	ld [wHPBarOldHP + 1], a
 	sbc b
 	ld [hl], a
-	ld [wHPBarNewHP+1], a
-	hlcoord 4, 1
+	ld [wHPBarNewHP + 1], a
+	hlcoord 11, 0
 	ld a, [wWhichPokemon]
 	ld bc, 2 * SCREEN_WIDTH
 	call AddNTimes ; calculate coordinates of HP bar of pokemon that used Softboiled
 	ld a, SFX_HEAL_HP
 	call PlaySoundWaitForCurrent
-	ldh a, [hUILayoutFlags]
-	set BIT_PARTY_MENU_HP_BAR, a
-	ldh [hUILayoutFlags], a
 	ld a, $02
 	ld [wHPBarType], a
 	predef UpdateHPBar2 ; animate HP bar decrease of pokemon that used Softboiled
-	ldh a, [hUILayoutFlags]
-	res BIT_PARTY_MENU_HP_BAR, a
-	ldh [hUILayoutFlags], a
 	pop af
 	ld b, a ; store heal amount (1/5 of max HP)
 	ld hl, wHPBarOldHP + 1
@@ -1214,7 +1231,7 @@
 	ld [hld], a
 	ld [wHPBarNewHP], a
 	ld a, [hl]
-	ld [wHPBarNewHP+1], a
+	ld [wHPBarNewHP + 1], a
 	jr nc, .noCarry
 	inc [hl]
 	ld a, [hl]
@@ -1251,7 +1268,7 @@
 	ld a, [hli]
 	srl a
 	ld [de], a
-	ld [wHPBarNewHP+1], a
+	ld [wHPBarNewHP + 1], a
 	ld a, [hl]
 	rr a
 	inc de
@@ -1263,7 +1280,7 @@
 .setCurrentHPToMaxHp
 	ld a, [hli]
 	ld [de], a
-	ld [wHPBarNewHP+1], a
+	ld [wHPBarNewHP + 1], a
 	inc de
 	ld a, [hl]
 	ld [de], a
@@ -1295,7 +1312,7 @@
 	xor a
 	ld [wBattleMonStatus], a ; remove the status ailment in the in-battle pokemon data
 .calculateHPBarCoords
-	hlcoord 4, -1
+	hlcoord 11, -2
 	ld bc, 2 * SCREEN_WIDTH
 	inc d
 .calculateHPBarCoordsLoop
@@ -1323,15 +1340,9 @@
 	jr z, .playStatusAilmentCuringSound
 	ld a, SFX_HEAL_HP
 	call PlaySoundWaitForCurrent
-	ldh a, [hUILayoutFlags]
-	set BIT_PARTY_MENU_HP_BAR, a
-	ldh [hUILayoutFlags], a
 	ld a, $02
 	ld [wHPBarType], a
 	predef UpdateHPBar2 ; animate the HP bar lengthening
-	ldh a, [hUILayoutFlags]
-	res BIT_PARTY_MENU_HP_BAR, a
-	ldh [hUILayoutFlags], a
 	ld a, REVIVE_MSG
 	ld [wPartyMenuTypeOrMessageID], a
 	ld a, [wCurItem]
@@ -1546,7 +1557,7 @@
 	pop af
 	ld [wWhichPokemon], a
 
-	callfar TryEvolvingMon
+	callfar TryEvolvingMon ; evolve pokemon, if appropriate
 	ld a, $01
 	ld [wUpdateSpritesEnabled], a
 	pop af
@@ -1556,12 +1567,16 @@
 	jp RemoveUsedItem
 
 VitaminStatRoseText:
-	text_far _VitaminStatRoseText
-	text_end
+	text_ram wNameBuffer
+	text "の　@"
+	text_ram wStringBuffer
+	text "の"
+	line "きそ　ポイントが　あがった！"
+	prompt
 
 VitaminNoEffectText:
-	text_far _VitaminNoEffectText
-	text_end
+	text "つかっても　こうかが　ないよ"
+	prompt
 
 INCLUDE "data/battle/stat_names.asm"
 
@@ -1619,12 +1634,12 @@
 	jp DelayFrames
 
 ThrewBaitText:
-	text_far _ThrewBaitText
-	text_end
+	text "<PLAYER>は　エサを　なげた！"
+	done
 
 ThrewRockText:
-	text_far _ThrewRockText
-	text_end
+	text "<PLAYER>は　いしを　なげた！"
+	done
 
 ; indirectly used by DIG in StartMenu_Pokemon.dig
 ItemUseEscapeRope:
@@ -1667,7 +1682,6 @@
 	ld c, 30
 	call DelayFrames
 	jp RemoveUsedItem
-
 .notUsable
 	jp ItemUseNotTime
 
@@ -1705,13 +1719,11 @@
 	jr nz, .next0
 	ld hl, CardKeyTable1
 	jr .next1
-
 .next0
 	cp $24
 	jr nz, .next2
 	ld hl, CardKeyTable2
 	jr .next1
-
 .next2
 	cp $5e
 	jp nz, ItemUseNotTime
@@ -1734,7 +1746,6 @@
 	ld a, [hl]
 	ld [wUnusedCardKeyGateID], a
 	jr .done
-
 .nextEntry1
 	inc hl
 .nextEntry2
@@ -1742,7 +1753,6 @@
 .nextEntry3
 	inc hl
 	jr .loop
-
 .done
 	ld hl, ItemUseText00
 	call PrintText
@@ -1985,15 +1995,18 @@
 	db -1 ; end
 
 PlayedFluteNoEffectText:
-	text_far _PlayedFluteNoEffectText
-	text_end
+	text "#のふえだ！"
+	line "ねむった#を　おこせるかも⋯"
+	prompt
 
 FluteWokeUpText:
-	text_far _FluteWokeUpText
-	text_end
+	text "すべての　#が"
+	line "めを　さました！"
+	prompt
 
 PlayedFluteHadEffectText:
-	text_far _PlayedFluteHadEffectText
+	text "<PLAYER>は"
+	line "#のふえを　ふいてみた！@"
 	text_promptbutton
 	text_asm
 	ld a, [wIsInBattle]
@@ -2020,8 +2033,11 @@
 	jp PrintText
 
 CoinCaseNumCoinsText:
-	text_far _CoinCaseNumCoinsText
-	text_end
+	text "あなたの　コイン"
+	line "@"
+	text_bcd wPlayerCoins, 2 | LEADING_ZEROES | LEFT_ALIGN
+	text "まい"
+	prompt
 
 ItemUseOldRod:
 	call FishingInit
@@ -2124,7 +2140,7 @@
 	ld a, SFX_HEAL_AILMENT
 	call PlaySound
 	ld a, $2
-	ld [wPikachuEmotionModifier], a
+	ld [wd49b], a
 	ld a, $81
 	ld [wPikachuMood], a
 	ld c, 80
@@ -2160,12 +2176,15 @@
 	jp PrintText
 
 ItemfinderFoundItemText:
-	text_far _ItemfinderFoundItemText
-	text_end
+	text "おッ！"
+	line "マシンが　はんのう　してるぞ！"
+	cont "ちかくに　アイテムが　うまってる！"
+	prompt
 
 ItemfinderFoundNothingText:
-	text_far _ItemfinderFoundNothingText
-	text_end
+	text "<⋯>　<⋯>　ふう！"
+	line "<⋯>　なんにも　はんのう　しない"
+	prompt
 
 ItemUsePPUp:
 	ld a, [wIsInBattle]
@@ -2373,24 +2392,31 @@
 	ret
 
 RaisePPWhichTechniqueText:
-	text_far _RaisePPWhichTechniqueText
-	text_end
+	text "どのわざの"
+	line "ポイントをふやす？"
+	done
 
 RestorePPWhichTechniqueText:
-	text_far _RestorePPWhichTechniqueText
-	text_end
+	text "どのわざを"
+	line "かいふくする？"
+	done
 
 PPMaxedOutText:
-	text_far _PPMaxedOutText
-	text_end
+	text_ram wStringBuffer
+	text "は　これいじょう"
+	line "ふやすことが　できません"
+	prompt
 
 PPIncreasedText:
-	text_far _PPIncreasedText
-	text_end
+	text_ram wStringBuffer
+	text "の"
+	line "わざポイントが　ふえた！"
+	prompt
 
 PPRestoredText:
-	text_far _PPRestoredText
-	text_end
+	text "わざポイントが"
+	line "かいふくした！"
+	prompt
 
 ; for items that can't be used from the Item menu
 UnusableItem:
@@ -2462,6 +2488,7 @@
 	call ClearSprites
 	call RunDefaultPaletteCommand
 	jp LoadScreenTilesFromBuffer1 ; restore saved screen
+
 .checkIfAbleToLearnMove
 	predef CanLearnTM ; check if the pokemon can learn the move
 	push bc
@@ -2507,7 +2534,7 @@
 	jr nz, .notTeachingThunderboltOrThunderToPikachu
 .teachingThunderboltOrThunderToPlayerPikachu
 	ld a, $5
-	ld [wPikachuEmotionModifier], a
+	ld [wd49b], a
 	ld a, $85
 	ld [wPikachuMood], a
 .notTeachingThunderboltOrThunderToPikachu
@@ -2520,20 +2547,36 @@
 	jp RemoveUsedItem
 
 BootedUpTMText:
-	text_far _BootedUpTMText
-	text_end
+	text "<TM>を　きどうした！"
+	prompt
 
 BootedUpHMText:
-	text_far _BootedUpHMText
-	text_end
+	text "ひでんマシンを　きどうした！"
+	prompt
 
 TeachMachineMoveText:
-	text_far _TeachMachineMoveText
-	text_end
+	text "なかには　@"
+	text_ram wStringBuffer
+	text "が"
+	line "きろくされていた！"
+
+	para "@"
+	text_ram wStringBuffer
+	text "を"
+	line "#に　おぼえさせますか？"
+	done
 
 MonCannotLearnMachineMoveText:
-	text_far _MonCannotLearnMachineMoveText
-	text_end
+	text_ram wNameBuffer
+	text "と　@"
+	text_ram wStringBuffer
+	text "は"
+	line "あいしょうが　わるかった！"
+
+	para "@"
+	text_ram wStringBuffer
+	text "は　おぼえられない！"
+	prompt
 
 PrintItemUseTextAndRemoveItem:
 	ld hl, ItemUseText00
@@ -2596,58 +2639,69 @@
 	jp PrintText
 
 ItemUseNotTimeText:
-	text_far _ItemUseNotTimeText
-	text_end
+	text "オーキドの　ことば<⋯>"
+	line "<PLAYER>よ！　こういうものには"
+	cont "つかいどきが　あるのじゃ！"
+	prompt
 
 ItemUseNotYoursToUseText:
-	text_far _ItemUseNotYoursToUseText
-	text_end
+	text "たいせつな　あずかりものです！"
+	next "つかうことは　できません！"
+	prompt
 
 ItemUseNoEffectText:
-	text_far _ItemUseNoEffectText
-	text_end
+	text "つかっても　こうかがないよ"
+	prompt
 
 ThrowBallAtTrainerMonText1:
-	text_far _ThrowBallAtTrainerMonText1
-	text_end
+	text "<TRAINER>に　ボールを　はじかれた！"
+	prompt
 
 ThrowBallAtTrainerMonText2:
-	text_far _ThrowBallAtTrainerMonText2
-	text_end
+	text "ひとの　ものを　とったら　どろぼう！"
+	prompt
 
 NoCyclingAllowedHereText:
-	text_far _NoCyclingAllowedHereText
-	text_end
+	text "ここでは　じてんしゃに"
+	next "のることは　できません"
+	prompt
 
 NoSurfingHereText:
-	text_far _NoSurfingHereText
-	text_end
+	text "ここでは@"
+	text_ram wNameBuffer
+	text "に"
+	line "のることは　できません"
+	prompt
 
 BoxFullCannotThrowBallText:
-	text_far _BoxFullCannotThrowBallText
-	text_end
+	text "ボックスに　あずけている　#が"
+	line "いっぱいなので　つかえません！"
+	prompt
 
 DontHavePokemonText:
-	text_far _DontHavePokemonText
-	text_end
+	text "#が　１ぴきも　いない！"
+	prompt
 
 ItemUseText00:
-	text_far _ItemUseText001
+	text "<PLAYER>は@"
 	text_low
-	text_far _ItemUseText002
-	text_end
+	text_ram wStringBuffer
+	text "を　つかった！"
+	done
 
 GotOnBicycleText:
-	text_far _GotOnBicycleText1
+	text "<PLAYER>は@"
 	text_low
-	text_far _GotOnBicycleText2
-	text_end
+	text_ram wStringBuffer
+	text "に　のった"
+	prompt
 
 GotOffBicycleText:
-	text_far _GotOffBicycleText1
+	text "<PLAYER>は@"
 	text_low
-	text_far _GotOffBicycleText2
-	text_end
+	text_ram wStringBuffer
+	text "から　おりた"
+	prompt
 
 ; restores bonus PP (from PP Ups) when healing at a pokemon center
 ; also, when a PP Up is used, it increases the current PP by one PP Up bonus
@@ -2878,16 +2932,21 @@
 	ret
 
 ThrewAwayItemText:
-	text_far _ThrewAwayItemText
-	text_end
+	text_ram wNameBuffer
+	text "を"
+	line "すてました！"
+	prompt
 
 IsItOKToTossItemText:
-	text_far _IsItOKToTossItemText
-	text_end
+	text_ram wStringBuffer
+	text "を　すてます"
+	line "ほんとに　よろしいですか？"
+	prompt
 
 TooImportantToTossText:
-	text_far _TooImportantToTossText
-	text_end
+	text "それは　とても　たいせつなモノです"
+	line "すてることは　できません！"
+	prompt
 
 ; checks if an item is a key item
 ; INPUT:
@@ -3132,7 +3191,7 @@
 	ld hl, ShoreTiles
 .skipShoreTiles
 	ld a, [wTileInFrontOfPlayer]
-	ld de, 1
+	ld de, $1
 	call IsInArray
 	ret
 
