--- intl/engine/battle/move_effects/substitute.asm
+++ jp/engine/battle/move_effects/substitute.asm
@@ -65,13 +65,16 @@
 	jp PrintText
 
 SubstituteText:
-	text_far _SubstituteText
-	text_end
+	text "<USER>の"
+	line "ぶんしんが　あらわれた"
+	prompt
 
 HasSubstituteText:
-	text_far _HasSubstituteText
-	text_end
+	text "しかし　<USER>の"
+	line "みがわりは　すでに　でていた！"
+	prompt
 
 TooWeakSubstituteText:
-	text_far _TooWeakSubstituteText
-	text_end
+	text "しかし　ぶんしんを　だすには"
+	line "たいりょくが　たりなかった！"
+	prompt
